values.yaml

```
@@ -12,6 +12,9 @@ image:
   # st2chatops and st2packs (which have their own override). This also does not impact 
   # dependencies such as mongo or redis, which have their own helm chart settings.
   repository: ""
+  # Image pull secret.
+  # May be required for public docker hub due to rate limiting or any private repository.
```

I think having a link to instructions about how to set up the secret to pull images from private docker registry will help here: 

```suggestion
  # May be required for public docker hub due to rate limiting or any private repository.
  # See instructions: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
```

---

Good idea - added it